### ISSUE TITLE
Warning on "docker orphans" in a ybt command

### ISSUE THREAD

**Issue by dlehavi-resonai** [resonai/ybt#103]
When you run ybt on an entire YBuild (or YBuilds), some of the targets in the YBuild may not end up in any docker. it may be better if this got a "target **** skipped" message, probably appearing at the very end of the YBT run.

**Comment by itamaro** [comment]
I think such "skipped" messages would be confusing, because the target wasn't actually skipped, it's just the maybe targets that depend on it (and use the artifacts from it) weren't built - which is ok, since the user didn't ask for that.

a related enhancement could be to "build what I asked for on the command line and also everything else in the project graph that depends on what I asked for in the command line" - would that solve the problem you're experiencing?

I think we have another issue for supporting more complex "graph queries", and this can be considered a subset of that.

thoughts?

**Comment by dlehavi-resonai** [comment]
I agree. much better. I think we should raise the priority on these graph thingies.
